gto-support-db/src/main/java/org/ccci/gto/android/common/db/Expression.java

```diff
@@ -760,4 +800,89 @@ public Unary createFromParcel(@NonNull final Parcel in, @Nullable final ClassLoa
             }
         }
     }
+
+    public static class Aggregate extends Expression {
+        static final String COUNT = "COUNT";
+        static final String MAX = "MAX";
+        static final String MIN = "MIN";
+        static final String SUM = "SUM";
+
+        @NonNull
+        private final String mOp;
+        @NonNull
+        private final Expression mExpr;
```

Comment:
I would also narrow this to `private final Field mExpr` since Aggregate functions can only apply to fields (I believe). we can widen this down the line if other use cases actually show up.